app/uat_actions/uat_actions.rb

```diff
@@ -48,6 +48,11 @@ def category
       CATEGORY_LIST.detect { |category| name.include?(category) } || CATEGORY_LIST.last
     end
 
+    # Returns a hash of all registered uat_actions grouped by category and sorted
+    def grouped_and_sorted_uat_actions(uat_actions)
+      uat_actions.group_by(&:category).sort_by { |category, _| UatActions::CATEGORY_LIST.index(category) }
```

Comment:
uat_actions should now be accessible in this scope so you may not need to pass it in. Similarly UatActions::CATEGORY_LIST can just become CATEGORY_LIST.